src/Microsoft.DotNet.Darc/src/DarcLib/AzureDevOpsClient.cs

```diff
@@ -286,13 +288,41 @@ public async Task<string> CreatePullRequestAsync(string repoUri, PullRequest pul
             VssConnection connection = CreateVssConnection(accountName);
             GitHttpClient client = await connection.GetClientAsync<GitHttpClient>();
 
+            // get the identity client used to resolve user name strings to identities
+            var identityClient = await connection.GetClientAsync<IdentityHttpClient>();
+
+            // Get the identities of each reviewer
+            var identities = new List<IdentityRefWithVote>();
+            foreach (var reviewer in pullRequest.Reviewers)
+            {
+                // look for a matching display name
+                var matchingIdentity = (await identityClient.ReadIdentitiesAsync(IdentitySearchFilter.DisplayName, reviewer).ConfigureAwait(false)).FirstOrDefault();
```

Comment:
There are a bunch of var's in this file that I don't know what they are referring to. Whta is "ReadIdentitiesAsync" retruning?  "var" is against our coding style.